js/runner_test.go

```diff
@@ -527,6 +527,7 @@ func TestVURunContext(t *testing.T) {
 				fnCalled = true
 
 				assert.Equal(t, vu.Runtime, common.GetRuntime(*vu.Context), "incorrect runtime in context")
+				assert.Nil(t, common.GetInitEnv(*vu.Context)) // shouldn't get this in the vu context
```

Comment:
It will be nice for there to be a test it is there in  the initcontext though, in the js packages, for example, bundle_test.go

Reply:
Yeah, that's one test I want to add - I couldn't do it here since `getSimpleRunner()` can't be used for it